frontend/src/citizen-frontend/LoginPage.tsx

```diff
@@ -36,12 +40,18 @@ export default React.memo(function LoginPage() {
 
   return (
     <>
-      <Container verticalMargin={defaultMargins.L}>
-        <FixedSpaceColumn spacing="L">
-          <ContentArea opaque paddingVertical="L">
+      <TabletAndDesktop>
+        <Gap size="L" />
+      </TabletAndDesktop>
+      <MobileOnly>
+        <Gap size="xs" />
```

Comment:
Was this supposed to be removed completely or just made extra small? Arguably a small gap looks good.

Reply:
It's 8 px on mobile in the Figma designs 👍